app/build.gradle

```diff
@@ -50,6 +50,7 @@ dependencies {
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.8'
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.5.2'
     implementation 'androidx.activity:activity-ktx:1.4.0'
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```

Comment:
Stop adding this dependency, it's not needed. If you are observant enough you would have noticed I removed it earlier. Create your fragments manually so it doesn't automatically add unnecessary dependencies to the project